packages/reakit/src/Toolbar/__examples__/ToolbarWithPopover/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { useToolbarState, Toolbar, ToolbarItem } from "reakit/Toolbar";
+import {
+  usePopoverState,
+  Popover,
+  PopoverDisclosure,
+  PopoverArrow,
+  PopoverProps,
+} from "reakit/Popover";
+import { Button } from "reakit/Button";
+
+const PopoverItem = React.forwardRef<HTMLButtonElement, PopoverProps>(
+  (_, ref) => {
+    const popover = usePopoverState();
+    return (
+      <>
+        <PopoverDisclosure {...popover} ref={ref}>
```

Comment:
We have to pass props to `PopoverDisclosure`. These props contain event handlers that will make this button behave as a toolbar item as well. I guess that's why using arrow keys while this button is focused isn't working.
```suggestion
  (props, ref) => {
    const popover = usePopoverState();
    return (
      <>
        <PopoverDisclosure {...popover} {...props} ref={ref}>
```